src/interface/api_swagger.json

```diff
@@ -3312,6 +3328,10 @@
           "description": "FSP ID (if known)",
           "minLength": 1,
           "maxLength": 32
+        },
+        "extensionList": {
```

Comment:
I think we should change the swagger version number to 1.0.1 to reflect the addition of the extensionList on PartyIdInfo